BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.27-prerelease-01321-01
+1.0.27-prerelease-01317-01
```

Comment:
Guess it moved a little too quick for me :)  I will update this and re-run the tests.